di-ipv-cimit-stub/deploy/template.yaml

```diff
@@ -138,6 +138,43 @@ Resources:
         - SSMParameterReadPolicy:
             ParameterName: stubs/core/cimit/*
 
+  # lambda to stub management
+  StubManagementFunction:
+    Type: AWS::Serverless::Function
+    # checkov:skip=CKV_AWS_109: this requires a broad set of permissions
+    DependsOn:
+      - "StubManagementFunctionLogGroup"
+    Properties:
```

Comment:
Also need to add "Events" which will trigger this function.

Reply:
will create a new API GW for that in the [PYIC-3214] and trigger from this API. So, we don't need here any event.